metashare/api/views.py

```diff
@@ -37,6 +37,25 @@ def get_object(self):
         return self.get_queryset().get()
 
 
+class CreatePrMixin:
+    error_pr_exists = ""  # Implement this
```

Comment:
I'm curious what this means. It looks like every class using `CreatePrMixin` already has `error_pr_exists` defined?

Reply:
I tend to put the attributes that a mixin needs on the attribute, with some kind of null value, and enjoin anyone using the mixin (typically just _me_) to set a real value for that attribute.

Reply:
Ok, that makes sense. Maybe I was confused because I was interpreting "Implement this" as a todo item.